packages/components/src/DataList/DataList.test.tsx

```diff
@@ -92,6 +92,16 @@ describe("DataList", () => {
       const results = screen.getByTestId(DATALIST_TOTALCOUNT_TEST_ID);
       expect(within(results).getByTestId(GLIMMER_TEST_ID)).toBeInTheDocument();
     });
+
+    it("should not render the total count if the totalCount is not declared", () => {
+      render(
+        <DataList data={mockData} headers={mockHeader} title={mockTitle}>
+          <></>
+        </DataList>,
+      );
+      expect(screen.getByText(mockTitle)).toBeInTheDocument();
+      expect(screen.queryByTestId(DATALIST_TOTALCOUNT_TEST_ID)).toBeNull();
```

Comment:
nit/not blocking; using `.not.toBeInTheDocument()` is, I think, slightly better to use here than `toBeNull()`. From a readability standpoint as well as semantically.  From what I understand, `toBeNull` is testing the value returned by `queryByTestId` whereas `.not.toBeInTheDocument` is checking for the actual absence of an element.

Reply:
Happy to make that change! I'll follow up with this change after we chat later today. Thanks 🚀